packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -0,0 +1,682 @@
+import { IsConformantOptions } from './types';
+import { ComponentDoc } from 'react-docgen-typescript';
+
+import chalk from 'chalk';
+import * as _ from 'lodash';
+import * as path from 'path';
+import parseDocblock from './utils/parseDocblock';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+
+// An array of all the failed isConformant tests for the component.
+const failedTests: string[] = [];
+
+export const defaultErrorMessages = {
+  'has-docblock': (componentInfo: ComponentDoc, testInfo: IsConformantOptions, error: string) => {
+    const { displayName, componentPath } = testInfo;
+    const fileName = path.basename(componentPath);
+    const docblock = parseDocblock(componentInfo.description);
+    const docBlockLength = _.words(docblock.description).length.toString();
+
+    // Message Description: Handles scenario where there is no existing docblock in the componentPath.
+    //
+    // It appears that "displayName" doesn't have a docblock in the file:: "componentPath".
+    // To Resolve this issue:
+    // 1. Consider adding a descriptive 5 to 25 word docblock in "fileName".
+    if (_.words(docblock.description).length === 0) {
+      console.log(
+        defaultErrorMessage(
+          `has-docblock`,
+          displayName,
+          'docblock in the file: ' + paragraph() + chalk.green.italic(componentPath),
+        ) +
+          resolveErrorMessages([
+            `Consider adding a descriptive` +
+              chalk.hex('#e00000')(' 5 ') +
```

Comment:
It might be best to stick with built-in colors, so that they adapt better to different terminal color schemes.

Reply:
On a related note, would be good to check the display of these messages in a light theme terminal if you're normally using dark (or vice versa).

Reply:
I investigated this and it appears that most console messages aren't readable in other color schemes. 
![image](https://user-images.githubusercontent.com/7192338/94973645-199f4680-04c1-11eb-94ab-747635b10b0d.png)
![image](https://user-images.githubusercontent.com/7192338/94973663-22901800-04c1-11eb-8e90-673f6430a571.png)

I decided to update the resolve issue text so it will be more readable in light mode but not sure what to do about the other color schemes.
![image](https://user-images.githubusercontent.com/7192338/94973690-30de3400-04c1-11eb-9bc6-34b627dc4b6d.png)

Reply:
Thanks for checking on that! I think the main concern should be the standard light and dark themes.

Reply:
One remaining related question, why did you use #e00000 instead of just red? And if you do want to keep using the custom color, it might be good to extract that part to a helper (under whatever name makes sense). `const otherRed = chalk.hex('#e00000'); ... otherRed(' 5 ')`

Reply:
It is a tiny detail. The color is slightly darker and intended to help differentiate between isConformant error messages and others that use the default red chalk. Also, I decided to extract all of colors into one errorMessageColor object to help keep consistency in any future added error messages.